templates/phoenix_elixir/files/config/config.exs

```diff
@@ -0,0 +1,30 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+config :template_elixir,
+  ecto_repos: [TemplateElixir.Repo]
+
+# Configures the endpoint
+config :template_elixir, TemplateElixirWeb.Endpoint,
+  url: [host: "localhost"],
```

Comment:
Eh?

Reply:
generated by the ` mix phx.new hello` command for creating a new app.